api/specs/webserver/openapi-publications.yaml

```diff
@@ -0,0 +1,35 @@
+paths:
+  /publications/service-submission:
+    post:
+      tags:
+        - publication
+      summary: Submits a new service candidate
+      operationId: service_submission
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              required:
+                - data
+              properties:
+                data:
+                  type: string
+                  format: binary
+                file:
+                  type: string
+                  format: binary
+      responses:
+        '200':
+          description: Submission has been registered
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - data
+                properties:
+                  data:
+                    type: boolean
```

Comment:
what means data as boolean?
is that for the envelope? in that case create an envelope. and in case the boolean is just to confirm and you do not need any data back return a 204 instead.

Reply:
done